tsconfig.json

```diff
@@ -10,11 +10,10 @@
     "preserveConstEnums": true,
     "removeComments": true,
     "resolveJsonModule": true,
-    "sourceMap": false,
+    "sourceMap": true,
     "strictNullChecks": true,
     "moduleResolution": "node",
-    "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true,
+    "esModuleInterop": false,
```

Comment:
Let's completely remove the line since that's the default (and I'm afraid when I see this in configs).